winter-website-contact-form.css

```diff
@@ -63,6 +63,60 @@ input[type=submit]:hover {
 	border: none;
 }
 
+@media screen and (max-width: 590px) {
+	
+	body {
+		min-width: 300px;
+	}
+	
+	.navBar {
+		display: flex;
+		flex-direction: column;
+		min-width: 30%;
+		width: 30%;
+		opacity: 1;
+		font-size: 80%;
+		padding: 5px 10px;
+		margin: 10px;
+		background-color: #856ab8;
+		padding: 0.5rem;
+		border-radius: 0.2rem;
```

Comment:
better practice I think to use px intead of rem for boreders, etc. Basically rem are used for font-size